kostyor/db/migrations/alembic/versions/2bedb47d68e6_initial_schema.py

```diff
@@ -0,0 +1,81 @@
+"""Initial database schema
+
+Revision ID: 2bedb47d68e6
+Revises: initial
+Create Date: 2016-09-09 11:02:40.083013
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2bedb47d68e6'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+STATUSES = ['READY FOR UPGRADE', 'UPGRADE IN PROGRESS', 'UPGRADE PAUSED',
+            'UPGRADE ERROR', 'UPGRADE CANCELLED', 'NOT READY FOR UPGRADE',
+            'UPGRADE ROLLBACK IN PROGRESS']
+
+VERSIONS = ['liberty', 'mitaka', 'newton', 'unknown']
```

Comment:
let's sync the order with open api spec https://github.com/sc68cal/Kostyor/pull/36/commits/6a62e365d76100d1d041e06196d815f0702e2327#diff-3550eb108976e83d6ab7269c59348b7fR155